src/AnchoredOverlay/AnchoredOverlay.tsx

```diff
@@ -90,6 +90,7 @@ export const AnchoredOverlay: React.FC<AnchoredOverlayProps> = ({renderAnchor, c
     },
     [overlayRef.current]
   )
+  const overlayPosition = position && {top: `${position.top}px`, left: `${position.left}px`}
```

Comment:
Worth a `React.useMemo`? Looks like most of the other props to `Overlay` are memoized.

Reply:
`useMemo` pushed @dmarcey.  LGTY?